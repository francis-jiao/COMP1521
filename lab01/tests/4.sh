"./$1" < tests/4.txt
