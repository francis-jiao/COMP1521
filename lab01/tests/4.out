abcdefghijklmnopqrstuvwayz
01234567890
abcdefghijklmnopqrstuvwayz
01234567890
abcdefghijklmnopqrstuvwayz
01234567890
